src/components/TypesOfRecipeCards/RecipeCard.jsx

```diff
@@ -3,13 +3,14 @@ import { Link } from "react-router-dom";
 import { pinWhite, pinSize } from "../../shared/style";
 
 const RecipeCard = ({ pinned, recipe }) => {
-  recipe = recipe.recipeId ? recipe.recipeId : recipe;
+  const review = recipe.recipeId;
+  const isRecipe = recipe.recipeId ? review : recipe;
```

Comment:
This is actually very confusing... If it has a recipeId, then it's a review... very strange